### PR TITLE
Fix for incorrect radius when exporting to SVG

### LINKED ISSUE #67: Border-radius values always '8' when exporting to SVG
When I set a border radius on a rectangle it renders as expected, however when I export the element as an SVG the border radius is always `8` unless I manually change it in Sketch. See screenshots below. I'm not sure if this would be a *react-sketchapp* issue though…

**How to reproduce:**
If you use the **basic-setup** example included with the repository: 

1. add a border radius to the Swatch:
````
<View
    name={`Swatch ${name}`}
    style={{
      height: 96,
      width: 96,
      margin: 4,
      backgroundColor: hex,
      padding: 8,
      borderRadius: 11,
    }}
  >
````

2. Choose one of the swatches and change the border radius manually.
3. Export as SVG and view the code.

**Sketch version:**
43.2

screenshot 2017-05-03 11 32 24
screenshot 2017-05-03 11 33 12
screenshot 2017-05-03 11 32 09
screenshot 2017-05-03 11 33 25

### PR DESCRIPTION
**Pull request by dchen**
Closes #67

### CODE REVIEW

**Comment by ljharb** [comment]
@dchen would you mind rebasing on the command line so there's no merge commits?

File: src/jsonUtils/shapeLayers.js [hunk 1]
```diff
@@ -132,32 +132,35 @@ export const makeRectShapeLayer = (
...
+    fixedRadius,
+    hasConvertedToNewRoundCorners: true,
+  };
+};
 
 export const makeShapeGroup = (
   frame: SJRect,
   layers: Array<any> = [],
-  fills?: Array<SJFill> = [],
+  fills?: Array<SJFill> = []
```
Comment (ljharb) [inline src/jsonUtils/shapeLayers.js]: our style guide requires trailing commas, on functions as well

**Comment by dchen** [comment]
I rebased.

Regarding the comma dangling, the pre-commit hook is removing that automatically. Is there some config I need to change to prevent that from happening?

**Comment by jongold** [comment]
Weird, I'll look into the config :/

On May 3, 2017 12:50 PM, "David E. Chen" <notifications@github.com> wrote:

I rebased.

Regarding the comma dangling, the pre-commit hook is removing that
automatically. Is there some config I need to change to prevent that from
happening?

—
You are receiving this because your review was requested.

Reply to this email directly, view it on GitHub
<https://github.com/airbnb/react-sketchapp/pull/69#issuecomment-299016285>,
or mute the thread

**Comment by ljharb** [comment]
I'm not sure; I generally avoid pre-commit hooks entirely since they aren't run in CI, and just like editor integrations, are only useful for devs themselves.

**Comment by dchen** [comment]
I updated my commit without running the pre-commit hook. Please take another look. Thanks!

**Pull request merged**